### ISSUE TITLE
Nested block groups .block-group-X-up class bug

### ISSUE THREAD

**Issue by Quetim** [Esri/calcite-web#659]
Nested **.block-group-Y-up .block** are *inheriting* top **.block-group-X-up** properties instead of the direct parent

## Example
```html
<div class="block-group block-group-3-up">
    <div class="block">
        <!-- Nested block group -->
        <div class="block-group block-group-2-up">
            <div class="block"> ... </div><!-- inherit block-group-3-up instead of 2 -->
            <div class="block"> ... </div><!-- inherit block-group-3-up instead of 2 -->
        </div>
    </div>
    <div class="block"> ... </div>
    <div class="block"> ... </div>
    <div class="block"> ... </div>
</div>
```

## Posible solution
```sass
.block-group-X-up{
    & > .block{
        // ...
    }
}
```
### Caveats
.block **should** be a direct children in order to work correctly

**Comment by paulcpederson** [comment]
@Quetim SGTM. I think block groups only work properly if they are direct decedents anyways, so this change shouldn't break anybody's project.

**Comment by paulcpederson** [comment]
addressed in #661 , will be part of next release
